k8s/wrappers/commands/k8s

```diff
@@ -4,26 +4,4 @@
 
 k8s::common::setup_env
 
-# TODO(neoaggelos): replace with an actual CLI, this is simply placeholder commands so that we can start iterating
-case "$1" in
-  init) k8s::init ;;
-  start) snapctl start k8s ;;
-  kubectl) shift; k8s::cmd::kubectl "${@}" ;;
```

Comment:
I discussed this with Angelos. We won't expose the kubectl binary for now since this causes issues like https://warthogs.atlassian.net/browse/KU-179.